src/main/java/com/bytebreak/animagic/BitTextureAtlas.java

```diff
@@ -0,0 +1,32 @@
+package com.bytebreak.animagic;
+
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.utils.Array;
+import com.badlogic.gdx.utils.ObjectSet;
+
+/**
+ * Created by Monday on 11/12/2015.
+ */
+public class BitTextureAtlas extends TextureAtlas {
+    public BitTextureAtlas(TextureAtlas.TextureAtlasData data) {
+        super(data);
+    }
+
+    public Array<TextureAtlas.AtlasRegion> findRegions(String name) {
```

Comment:
This piece is the 'magic'. It just hides the fact that we are actually scanning a directory for numbered images. Now we can have one directory per animation to keep clutter down (theoretically)